app/src/main/java/com/example/harvestfresh/AlarmReceiver.java

```diff
@@ -0,0 +1,39 @@
+package com.example.harvestfresh;
+
+import android.app.Notification;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+public class AlarmReceiver extends BroadcastReceiver {
+
+    private final int INTENT_CODE = 0;
+    private final int NOTIFY_ID = 123;
+    private final String CHANNEl_ID = "HarvestFresh";
+    private final String NOTIFY_TITLE = "HarvestFresh - Your Cart is Waiting";
+    private final String NOTIFY_TEXT = "Your cart has been idle for 20 minutes, if you don't checkout it will be deleted soon.";
```

Comment:
Protip: I dont think this is hooked up to anything, but when you do, you're going to have a const for "20" somewhere. You can reference it here when you do, so this string automatically updates when you change it.
has been idle for " + IdleTimer.IDLE_TIME_MINUTES + ", if you don't checkout